examples/gridViewExample/build.gradle

```diff
@@ -26,6 +26,14 @@ android {
     command {
         events 2000
     }
+
+    splits {
+        // Split apks on build target version
+        abi {
+            enable true
+            include 'arm', 'arm-v7a', 'arm64', 'mips', 'x86', 'x86_64'
+        }
```

Comment:
The `universalApk` setting is a bit orthogonal to the idea of splitting the APK to reduce the size of the file. So I think it is fair to leave it out in the example. Maybe we should add a link to the API docs in the sample, as it is documented there?